### PR TITLE
Search tweaks

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

**Comment by AndrewBestbier** [comment]
@Smeb, you'll notice the million commits I have made. This is because we can't view the search page results unless they are deployed on live. I could squash them into one, but I think it is fine for now

File: styles/_blog.less [hunk 1]
```diff
@@ -230,3 +253,173 @@ article {
...
+  background-color: white;
+  -webkit-appearance: none;
+}
+
+.sqs-spin.large {
+  height: 12px;
+  width: 12px;
+}
+
+.sqs-search-ui-button-wrapper .search-input::placeholder {
```
Comment (AndrewBestbier) [inline styles/_blog.less]: delete

File: styles/_blog.less [hunk 2]
```diff
@@ -230,3 +253,173 @@ article {
...
+  display: none;
+}
+
+.sqs-search-preview-ui .sqs-search-ui-result .sqs-search-ui-list .search-result.clear {
+  margin: 0px;
+  padding: 0px;
+}
+
+input::-webkit-input-placeholder {
+  color: #a9a9a9 !important;
```
Comment (AndrewBestbier) [inline styles/_blog.less]: Are these !importants needed

File: styles/_blog.less [hunk 3]
```diff
@@ -230,3 +253,173 @@ article {
...
+  top: 40%;
+}
+
+
+.sqs-search-preview-ui .sqs-search-ui-result .sqs-search-ui-list .search-result.clear:after {
+  content:"";
+  bottom: 0;
+  left: 0;
+  margin-left: 65px;
+  margin-right: 65px;
```
Comment (AndrewBestbier) [inline styles/_blog.less]: margin: 0px 65px

File: styles/_blog.less [hunk 4]
```diff
@@ -230,3 +253,173 @@ article {
...
+  font-weight: bold;
+  padding-top: 10px;
+  padding-left: 0px;
+}
+
+
+
+@media (min-width: 1200px) {
+  .sqs-search-preview-ui .sqs-search-ui-result .sqs-search-ui-list .search-result {
+    margin-left: 95px;
```
Comment (AndrewBestbier) [inline styles/_blog.less]: margin: 0px 95px

File: styles/_blog.less [hunk 5]
```diff
@@ -167,6 +166,30 @@ article {
...
+  font-weight: 800;
+  color: #212121;
+  font-size: 30px;
+  border-bottom: 2px solid transparent;
+  display: inline-block;
+}
+
+
+.sqs-search-container-item .sqs-title:hover {
+  border-bottom: 2px solid #fc1d42;
```
Comment (asavin) [inline styles/_blog.less]: color variable instead?

File: styles/_blog.less [hunk 6]
```diff
@@ -167,6 +166,30 @@ article {
...
+}
+
+.sqs-search-container-item .sqs-content {
+  color: #212121;
```
Comment (asavin) [inline styles/_blog.less]: color variable instead?

File: styles/_blog.less [hunk 7]
```diff
@@ -230,3 +253,173 @@ article {
...
+.sqs-search-container-item .sqs-main-image-container {
+  display: none;
+}
+
+.sqs-search-preview-ui .sqs-search-ui-result .sqs-search-ui-list .search-result.clear {
+  margin: 0px;
+  padding: 0px;
+}
+
+input::-webkit-input-placeholder {
```
Comment (asavin) [inline styles/_blog.less]: this one will affect input fields everywhere on the whole site. Is this the intention?

File: styles/_blog.less (hunk 5 above)
Comment (AndrewBestbier) [inline styles/_blog.less]: Can do

File: styles/_blog.less (hunk 6 above)
Comment (AndrewBestbier) [inline styles/_blog.less]: Can do

File: styles/_blog.less (hunk 7 above)
Comment (AndrewBestbier) [inline styles/_blog.less]: Yep.

**Comment by lazydevorg** [comment]
👍 LGTM

**Pull request merged**